examples/client/client.c

```diff
@@ -1017,6 +1228,15 @@ THREAD_RETURN WOLFSSH_THREAD client_test(void* args)
     if (ssh == NULL)
         err_sys("Couldn't create wolfSSH session.");
 
+#ifdef WOLFSSH_TPM
+    /* Link the TPM 2.0 device to the SSH session
+     * This way, SSH client operations that can
+     * take advantage of the hardware security
+     * will have access to the TPM 2.0 device */
+    wolfSSH_SetTpmDev(ssh, &tpmDev);
```

Comment:
Would a single API for both be easier?

Reply:
Initially, I thought like this also. Then, realized there would be an instance, when tpmDev is linked and only the tpmKey used is changed or has to be changed. 

Image the use case

Customer's wolfSSH app
- Init TPM
- Set tpmDEv
- Load Key
- Set tpmKey
- Use Key
- ......
- Unload handle, Load another Key
- Set **new** tpmKey
===== and we can not be sure tpmKey would be exposed in this scenario, so setting just the tpmKey makes sense.

IF, I add GetTpmDev, then Set could be a single call for both device and key

SetTpm(tpmDev, TpmKey)
... user operations ...
GetTpmDev(&tpmDev);
SetTpm(tpmDev, NewTpmKey);